web/impact/impact/v1/views/base_list_view.py

```diff
@@ -120,6 +123,10 @@ def results(self, limit, offset):
         queryset = self.helper_class.all_objects()
         queryset = self.filter(queryset)
         count = queryset.count()
```

Comment:
I notice in functional testing that this returns the total count of objects, not the count of objects as limited by any filtering. So for example if I add the filter`?round_type=Online` I would expect to get a count of 25 (since that's how many judging rounds exist in the clean db with that round_type) but instead I get a count of 73. 

Since this is not your code, this is not a blocking issue - if you don't want to address this here, lmk and I'll create a bug about it. But it would probably be nice to get it sorted here.